File: src/test/java/tafqeetj/HundredsTest.java
package tafqeetj;

import org.junit.Test;
import tafqeetj.converters.Tafqeet;

public class HundredsTest {
    private final Tafqeet tafqeet = Tafqeet.getInstance();

    @Test
    public void test100NumberToText() {
        int num = 100;
        String result = this.tafqeet.doTafqeet(num);
        assert("مئة".equals(result));
    }

    @Test
    public void test200NumberToText() {
        int num = 200;
        String result = this.tafqeet.doTafqeet(num);
        assert("مئتين".equals(result));
    }

    @Test
    public void test300NumberToText() {
        int num = 300;
        String result = this.tafqeet.doTafqeet(num);
        assert("ثلاثمئة".equals(result));
    }

    @Test
    public void test400NumberToText() {
        int num = 400;
        String result = this.tafqeet.doTafqeet(num);
        assert("أربعمئة".equals(result));
    }

    @Test
    public void test500NumberToText() {
        int num = 500;
        String result = this.tafqeet.doTafqeet(num);
        assert("خمسمئة".equals(result));
    }

    @Test
    public void test600NumberToText() {
        int num = 600;
        String result = this.tafqeet.doTafqeet(num);
        assert("ستمئة".equals(result));
    }

    @Test
    public void test700NumberToText() {
        int num = 700;
        String result = this.tafqeet.doTafqeet(num);
        assert("سبعمئة".equals(result));
    }

    @Test
    public void test800NumberToText() {
        int num = 800;
        String result = this.tafqeet.doTafqeet(num);
        assert("ثمانمئة".equals(result));
    }

    @Test
    public void test900NumberToText() {
        int num = 900;
        String result = this.tafqeet.doTafqeet(num);
        assert("تسعمئة".equals(result));
    }
}

File: src/test/kotlin/tech/moussa/tafqeetk/HundredsTest.kt
package tech.moussa.tafqeetk

import org.junit.Test
import tech.moussa.tafqeetk.converters.Tafqeet

class HundredsTest {
    private val tafqeet: Tafqeet = Tafqeet.getInstance()

    @Test
    fun test100NumberToText() {
        val num = 100
        val result = this.tafqeet.doTafqeet(num)
        assert("مئة" == result)
    }

    @Test
    fun test200NumberToText() {
        val num = 200
        val result = this.tafqeet.doTafqeet(num)
        assert("مئتين" == result)
    }

    @Test
    fun test300NumberToText() {
        val num = 300
        val result = this.tafqeet.doTafqeet(num)
        assert("ثلاثمئة" == result)
    }

    @Test
    fun test400NumberToText() {
        val num = 400
        val result = this.tafqeet.doTafqeet(num)
        assert("أربعمئة" == result)
    }

    @Test
    fun test500NumberToText() {
        val num = 500
        val result = this.tafqeet.doTafqeet(num)
        assert("خمسمئة" == result)
    }

    @Test
    fun test600NumberToText() {
        val num = 600
        val result = this.tafqeet.doTafqeet(num)
        assert("ستمئة" == result)
    }

    @Test
    fun test700NumberToText() {
        val num = 700
        val result = this.tafqeet.doTafqeet(num)
        assert("سبعمئة" == result)
    }

    @Test
    fun test800NumberToText() {
        val num = 800
        val result = this.tafqeet.doTafqeet(num)
        assert("ثمانمئة" == result)
    }

    @Test
    fun test900NumberToText() {
        val num = 900
        val result = this.tafqeet.doTafqeet(num)
        assert("تسعمئة" == result)
    }
}